src/main/java/seedu/duke/storage/PatientList.java

```diff
@@ -0,0 +1,36 @@
+package seedu.duke.storage;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import seedu.duke.record.Patient;
+
+public class PatientList {
+    private List<Patient> patientList;
+
+    //constructor if no existing save file, create new list
+    public PatientList(){
+        this.patientList = new ArrayList<>();
+    }
+
+    //constructor if there is existing save file
+    public PatientList(List<Patient> savedPatientList) {
+        this.patientList = savedPatientList;
+    }
+
+    public List<Patient> getPatientList() {
+        return this.patientList;
+    }
+
+    public void setPatientList(List patientListInput) {
+        this.patientList = patientListInput;
+    }
+
+    public Patient getPatientRecord(int patientIndex) {
+        return this.getPatientList().get(patientIndex);
+    }
+
+    public int getTotalPatients() {
+        return this.patientList.size();
+    }
```

Comment:
Thanks Brandon  💯. What do you think of making getTotalPatients (or even setters/getters methods) static so there won't be a need to create an instance of PatientList when we want to call these methods?

Reply:
However, it is ok to keep it this way, but take note of the constructor at line 12 - 14. Because we have to construct this object to call these methods, the default constructor from 12-14 will erase the existing patientList we have. SOoooo this might be a problem in the future~